src/scripts/Publish-Toolkit.ps1

```diff
@@ -95,7 +95,7 @@ function Find-Repo($config, [string]$templateName)
 $ver = & "$PSScriptRoot/Get-Version.ps1"
 
 # Loop thru templates
-Get-ChildItem "$relDir/$Template*" -Directory `
+Get-ChildItem "$relDir/$Template*" -Directory -Exclude "optimization-engine" `
```

Comment:
Should we add a `# TODO` comment above this line to remove the exclusion after we determine how we're going to publish AOE to other repos? I don't think we want this to be the end state.